jobset-generator/default.nix

```
@@ -0,0 +1,11 @@
+{ pkgs }:
+pkgs.poetry2nix.mkPoetryApplication {
+  projectDir = ./src;
+
+  meta = with pkgs.lib; {
+    homepage = "https://github.com/DeterminateSystems/hydra-github-project";
+    description = "Declaratively generate jobsets for all a project's branches and PRs.";
+    license = licenses.mit;
+    maintainers = teams.DeterminateSystems;
```

Prior to this suggestion, this won't actually evaluate:

1) `teams.DeterminateSystems` does not exist
2) `meta.maintainers` expects a list of people, while `teams.determinatesystems` is an attrset of `members` and `scope`

```suggestion
    maintainers = teams.determinatesystems.members;
```